src/components/accession2/withdraw/WithdrawModal.tsx

```diff
@@ -327,6 +347,32 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
     onClose();
   };
 
+  const onChangeWithdrawBy = (_: React.ChangeEvent<HTMLInputElement>, value: string) => {
+    if (value === 'count') {
+      setIsByWeight(false);
+      setRecord({
+        ...record,
+        withdrawnQuantity: {
+          quantity: record.withdrawnQuantity?.quantity ? record.withdrawnQuantity?.quantity : 0,
+          units: 'Seeds',
+        },
+      });
+      setIndividualError('withdrawnQuantity', '');
```

Comment:
Clear this at the beginning of the function (applying to both value cases)? Line 370 sets the error again if relevant.